### DISCUSSION TITLE
Why does the If activity in Elsa 3.1.3 designer only have the Condition attribute and not the Then and Else attributes?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5679] [Design & tech]
Elsa 3.1.3 designer seems to have an issue where Then and Else properties are not visible in the designer interface.
![If](https://github.com/elsa-workflows/elsa-core/assets/171897414/7554fa7b-ca12-488b-981c-3cccdee819ed)

**Comment by glime-ai** [comment]
Based on the context provided, it appears that there is a known issue with the Elsa 3.1.3 designer where the Then and Else properties are not visible for the If activity. This seems to be a bug or a missing feature in the current version of the designer.

To address this issue, you may want to check the official Elsa GitHub repository for any updates or patches that have been released to fix this problem. Additionally, you can report the issue on the repository if it hasn't been reported yet, or check if there are any workarounds suggested by the community.

For further assistance, you can also reach out to the Elsa community on their Discord channel or Stack Overflow with the tag `elsa-workflows` to see if other users have encountered the same issue and have found a solution.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
